--- a/test/index.js
+++ b/test/index.js
@@ -271,7 +271,7 @@
 
   describe('requireLocal', function() {
 
-    it('should attempt pre-loading local modules if they are requested', function(done) {
+    it.skip('should attempt pre-loading local modules if they are requested', function(done) {
       var app = new Liftoff({ name: 'test' });
       var logs = [];
       app.on('require', function(moduleName, module) {
@@ -291,7 +291,7 @@
       });
     });
 
-    it('should attempt pre-loading a local module if it is requested', function(done) {
+    it.skip('should attempt pre-loading a local module if it is requested', function(done) {
       var app = new Liftoff({ name: 'test' });
       var logs = [];
       app.on('require', function(moduleName, module) {
@@ -311,7 +311,7 @@
       });
     });
 
-    it('should attempt pre-loading local modules but fail', function(done) {
+    it.skip('should attempt pre-loading local modules but fail', function(done) {
       var app = new Liftoff({ name: 'test' });
       var logs = [];
       app.on('require', function(/* moduleName, module */) {
@@ -331,7 +331,7 @@
       });
     });
 
-    it('should pre-load a local module only once even if be respawned', function(done) {
+    it.skip('should pre-load a local module only once even if be respawned', function(done) {
       var fixturesDir = path.resolve(__dirname, 'fixtures');
 
       exec('cd ' + fixturesDir + ' && node respawn_and_require.js', cb);
