Description: fix for node-is-plain-object ≥ 5
Author: Xavier Guimard <yadd@debian.org>
Forwarded: no
Last-Update: 2020-12-15

--- a/index.js
+++ b/index.js
@@ -5,7 +5,7 @@
 var extend = require('extend');
 var resolve = require('resolve');
 var flaggedRespawn = require('flagged-respawn');
-var isPlainObject = require('is-plain-object');
+var { isPlainObject } = require('is-plain-object');
 var mapValues = require('object.map');
 var fined = require('fined');
 
--- a/package.json
+++ b/package.json
@@ -26,7 +26,7 @@
     "findup-sync": "^3.0.0",
     "fined": "^1.0.1",
     "flagged-respawn": "^1.0.0",
-    "is-plain-object": "^2.0.4",
+    "is-plain-object": "^5.0.0",
     "object.map": "^1.0.0",
     "rechoir": "^0.6.2",
     "resolve": "^1.1.7"
